### DISCUSSION TITLE
5 weeks 11.000 km touring experience

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#594] [Show and tell]
I would like to share my 2024 5-week motorcycle tour experience with Cruiser here (7,000 km trip to Greece in April and 4,000 km trip to Corsica in June). 
I planned the routes at home on the desktop computer with Kurviger-Web, exported to Cruiser as *.kurviger file. I made all necessary route adjustments on the go with the Cruiser app. 
I did 100% of the navigation with Cruiser (recalculation Off, No voice guidance) and it worked perfectly for me, couldn't be better 🙂. 
My navigation setup consists of a Google Pixel 6 (exclusively as a sat nav) with SP Connect wireless charger. Both worked reliably. However the Pixel 6 stops inductive charging above 25 degrees (device protection?). Didn't cause any problems during either tour, but could be annoying in midsummer. 
Two wishes for Cruiser: 
-Refine the routing options to make them as powerful (and easy to use) as in Kurviger.
-Keep the risk of over-complexity at bay.

Thank you for the excellent work👏

**Comment by devemux86** [comment]
@GuG1050 thank you for sharing your experience with Cruiser!

**Reply by 1maxi2** [reply]
-Halten Sie das Risiko einer Überkomplexität in Schach.👍
